nri/index.html

```diff
@@ -0,0 +1,232 @@
+        </a></div>
+       <div class="icon-item"><a href="#"><img src="images/icon-linkedin.svg" alt="Linkedin">
+        </a></div>
+      </div>
+      <div class="article-content">
+        <h2 class="article-content-title"><img src="images/heading_icon.PNG" alt="icon">NRIのデジタルビジネス戦略</h2>
+        <div class="article-content-text">
+        <p>NRI では、企業の DX に向けた取り組みについて、既存ビジネスの業務プロセス変革に寄与する DX を「DX1.0」、デジタルで新しいビジネスモデルそのものを生み出す DX を「DX2.0」と定義しています。</p>
+        <p>DX を実現するためには、業務プロセス変革やビジネスモデル創造、そして、それらを実現する IT を同時並行で、共に議論しながら、かつ、スピーディーに実装できる戦略パートナーが必要です。NRI は、コンサルティングと IT ソリューションの人材と機能を併せ持ち、お客さまと併走し、お客さまと仮説検証を繰り返しながらビジネスを創出する「コンソリューション」の提供を強みとして、DX の実現を推進します。</p>
+        </div>
+      </div>
+      <div class="article-content-img"><img src="images/story06_img_01.jpg" alt="NRIのデジタルビジネス戦略"></div>
+      <div class="article-content-pdf"><a href="#">NRIのデジタルトランスフォーメーション（7.43 MB）<img src="images/pdf.png" alt="NRIのデジタルトランスフォーメーション"></a></div>
+      <ul class="article-content-page_frontback">
+        <li><p>＜<a href="#">ビジネスモデルの特徴</a></p></li>
+        <li><p><a href="#">人材が支えるNRI</a>></p></li>
+      </ul>
+      <div class="article-content-page_block">
+        <h4>NRIの強み</h4>
+        <div class="container">
+          <div class="item"><a href="#">NRIのあゆみと持続的な成長</a></div>
+          <div class="item"><a href="#">問題発見から問題解決まで</a></div>
+          <div class="item"><a href="#">強固な顧客基盤</a></div>
+          <div class="item"><a href="#">ビジネスモデルの特徴</a></div>
+          <div class="item"><a href="#">人材が支えるNRI</a></div>
+          <div class="item"><a href="#">業界トップクラスの収益力</a></div>
+        </div>
+      </div>
+      <div class="article-icon_below">
+        <p>Share</p>
+        <div class="icon-item"> <a href="#"><img src="images/icon-facebook.svg" alt="Facebook"></a>
+        </div>
+        <div class="icon-item"><a href="#"><img src="images/icon-twitter.svg" alt="Twitter"></a>
+        </div>
+        <div class="icon-item"><a href="#"><img src="images/icon-linkedin.svg" alt="Linkedin"></a>
+        </div>
+      </div>
+    </article>
+
+    <footer>
+      <div class="corporate_info">
+        <h2>企業情報</h2>
+        <div class="Management policy">
+          <div class="container">
+            <h4>経営方針</h4>
+            <div class="item"><a href="#">社長メッセージ</a></div>
+            <div class="item"><a href="#">企業理念</a></div>
+            <div class="item"><a href="#">経営ビジョン</a></div>
+            <div class="item"><a href="#">品質向上・リスク管理</a></div>
+            <div class="item"><a href="#">人材の育成</a></div>
+            <div class="item"><a href="#">内部統制</a></div>
+            <div class="item"><a href="#">NRIグループ企業行動原則</a></div>
+            <div class="item"><a href="#">NRIグループビジネス行動基準</a></div>
+            <div class="item"><a href="#">コーポレート・ガバナンス</a></div>
+            <div class="item"><a href="#">事業等のリスク</a></div>
+            <div class="item"><a href="#">NRIグループの調達方針</a></div>
```

Comment:
このあたりの要素は、「経営方針」という要素の箇条書きという理解が一番正しいです。
ですので、下記のような形にしてください。（以下同じ指摘）。

HTML5以前は、適切にマークアップできるタグが少なかったので`div`を多様しましたが
現在はできるだけ`div`を使わず（使っていけないわけではない）に、きちんと意味あるマークうアップをするのがデファクトスタンダードです。

まずは意味のあるマークアップができないか？を考えて、だめならdivを使うという順番で考えると良いです。

こういう考えをセマンティックwebといいますね。
see: [セマンティックweb](https://www.web-meister.jp/guide/glossary/glossary_sa/semantic_web.html#:~:text=%E3%82%BB%E3%83%9E%E3%83%B3%E3%83%86%E3%82%A3%E3%83%83%E3%82%AFWeb%E3%81%A8%E3%81%AF%E4%BB%8A,%E4%BB%95%E7%B5%84%E3%81%BF%E3%81%AE%E3%81%93%E3%81%A8%E3%81%A7%E3%81%82%E3%82%8B%E3%80%82)

```suggestion
          <ul>
            <li class="item"><a href="#">社長メッセージ</a></li>
            <li class="item"><a href="#">企業理念</a></li>
            ...以下略
          </ul>
```